test/test_spherical_advection.jl

```diff
@@ -34,23 +34,47 @@ EXAMPLES_DIR = pkgdir(TrixiAtmo, "examples")
     end
 end
 
-@trixiatmo_testset "Spherical advection, Cartesian form with element-local mapping" begin
```

Comment:
The test "Spherical advection, Cartesian form with element-local mapping" got lost at some point. Is this intentional?

Reply:
Nope, it was a mistake due to them being different PRs. I can add it back in!

Reply:
The new tests now have both, as of [b7a7f5a](https://github.com/trixi-framework/TrixiAtmo.jl/pull/31/commits/b7a7f5a6d7e4443d08b30020470da555e273188a).